lab-trent/index.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const http = require('http');
+
+const requestParser = require('./lib/requestparser');
+const cowsayRes = require('./lib/cowsayres');
+
+const PORT = process.env.PORT || 3000;
+
+const server = http.createServer(function(req, res) {
+  requestParser.parseUrlData(req);
+  requestParser.parseBody(req, function(body) {
+    req.body = body;
+    if (req.url.pathname === '/') {
```

Comment:
I would also like to have you validate the HTTP Method before the route. It'll help clean up the code a bit.